config/locales/client.en.yml

```
@@ -4701,6 +4701,7 @@ en:
         download: Download
         clear_all: Clear All
         clear_all_confirm: "Are you sure you want to clear all watched words for the %{action} action?"
+        invalid_regex: 'The watched word %{word} is an invalid regular expression.'
```

I think we should surround the regex with double quotes to ensure it's easily identifiable

```suggestion
        invalid_regex: 'The watched word "%{word}" is an invalid regular expression.'
```